AxonDeepSeg/apply_model.py

```diff
@@ -222,12 +258,12 @@ def axon_segmentation(path_acquisitions_folders, acquisitions_filenames, path_mo
 
     # Generating the patch to acquisitions and loading the acquisitions resolutions.
     path_acquisitions = [path_acquisitions_folders[i] / e for i, e in enumerate(acquisitions_filenames)]
-
+    print(path_acquisitions)
```

Comment:
This appears to be a debugging print that hasn't been deleted yet. If you actually want to print this information for the user, it should be a little bit more verbose (i.e. include accompanying text explaining what you are printing)

Reply:
Yes this was a debugging print. I will fix this.

Reply:
Resolved in c605fdded